data/rulesets.ts

```diff
@@ -861,7 +861,10 @@ export const BattleFormats: {[k: string]: FormatsData} = {
 		desc: "Prevents Pok&eacute;mon on the Ubers dynamax banlist from dynamaxing",
 		onBegin() {
 			for (const pokemon of this.getAllPokemon()) {
-				if (this.ruleTable.isRestricted('pokemon:' + pokemon.species.id)) pokemon.canDynamax = false;
+				if (this.ruleTable.isRestricted(`pokemon:${pokemon.species.id}`) ||
+					this.ruleTable.isRestricted(`basepokemon:${pokemon.species.id}`)) {
+					pokemon.canDynamax = false;
```

Comment:
are you sure its not this @Kris? this seems changed as well?

Reply:
This was also a fix (and my @ is not kris), i was pointing out that the fix was adding `basepokemon` checks in general